modules/Schedulers/Scheduler.php

```diff
@@ -398,11 +398,8 @@ function deriveDBDateTimes($focus)
 		//_pp($hrName);
 		// derive minutes
 		//$currentMin = date('i');
-        $minName = array();
-		$currentMin = $timedate->getNow()->minute;
-		if(substr($currentMin, 0, 1) == '0') {
-			$currentMin = substr($currentMin, 1, 1);
-		}
+        	$minName = array();
+		$currentMin = $timedate->getNow()->min*1;
```

Comment:
It would be better to cast the type instead. As this would make it more readable to developers in the future. 
`$currentMin = (int) $timedate->getNow()->min`